src/unix/linux_like/linux/gnu/b32/csky/align.rs

```diff
@@ -0,0 +1,7 @@
+s_no_extra_traits! {
+    #[allow(missing_debug_implementations)]
+    #[repr(align(2))]
+    pub struct max_align_t {
+        priv_: [i8; 20]
```

Comment:
Is the size supposed to be significantly larger than the alignment or was it meant to be 2 too?

Reply:
I will fix it, It's my ignorance.